Standards/scs-xxxx-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -101,7 +109,8 @@ Every list of standards consists of several standards that – altogether – de
 |-----	|------	|-------------	|---------- |
 | `{layer}.versions.standards.name` | String | Full name of the particular standard | _Flavor naming_ |
 | `{layer}.versions.standards.url` | String |  Valid URL to the latest raw version of the particular standard  | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Standards/SCS-0003-v1-flavor-naming.md_ |
-| `{layer}.versions.standards.check_tool` | String | Valid URL to the latest check tool that verifies compliance with the particular standard | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Design-Docs/tools/flavor-name-check.py_ |
+| `{layer}.versions.standards.check_tool` | String | Valid local filename or URL to the latest check tool that verifies compliance with the particular standard | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Design-Docs/tools/flavor-name-check.py_ |
```

Comment:
Should we refactor this to an array of `{layer}.versions.{version}.standards.check_tool.url` and `{layer}.versions.{version}.standards.check_tool.args`?

If we ever have to – for whatever reason – add multiple check tooks for a standard, we could simply append further tools to the overall YAML key {layer}.versions.{version}.standards.check_tool`.

Reply:
Having the ability to list multiple tools has the advantage potentially saving us a wrapper that calls multiple checking tools.
```yaml
iaas:
  - version: v2
    stabilized_at: 2525-02-25
    standards:
      - name: Flavor naming
        url: https://...
        check_tool:
          - url: filename_or_url
            args: -v
          - url: filename2
```
I have no strong opinion on this.

Reply:
In case it's not obvious: The logic of course would be that all tools need to pass, i.e. we just add the errors.

Reply:
See branch feat/compliance-tool-drop-cersions-add-tool-array.
Note that I renamed check_tool to check_tools to indicate it's an array.

Remaining question: Is the condition flags per check tool or per standard?
If for example a standard gets enhanced with additional requirements, I could image that the new properties are being tested in a new tool. In v1, we might still call this new tool with the optional condition, generating a report that allows us to understand how far we have to still go. In v2, it would then switch to mandatory ...